base/applications/notepad/dialog.c

```diff
@@ -945,6 +944,7 @@ VOID DoShowHideStatusBar(VOID)
     ShowWindow(Globals.hStatusBar, (Globals.bShowStatusBar ? SW_SHOWNOACTIVATE : SW_HIDE));
 
     /* Update status bar contents */
+    DIALOG_StatusBarAlignParts();
```

Comment:
This is not needed. `WM_SIZE` message above have done it already.

https://github.com/reactos/reactos/blob/9672dc50479f42c6da51b426618f05a9394a9ef2/base/applications/notepad/dialog.c#L933

https://github.com/reactos/reactos/blob/9672dc50479f42c6da51b426618f05a9394a9ef2/base/applications/notepad/main.c#L389

Reply:
In order to set each part's text, the parts must be already split.

Reply:
I will change "posting" to "sending".

Reply:
Fixed.